example/sriov-network/infiniband/sriov-network-pod1.yaml

```diff
@@ -30,5 +30,7 @@ spec:
           nvidia.com/mlnxics: "1"
         limits:
           nvidia.com/mlnxics: "1"
-#  nodeSelector:
-#    kubernetes.io/hostname: <worker-node-name>
+  nodeSelector:
+    feature.node.kubernetes.io/pci-15b3.sriov.capable: "true"
```

Comment:
why is this selector needed ?

Reply:
it was added to be more explicit. Let's delete is, since it's not required